### PR TITLE
docs: add a Getting Started section to the README

### PR DESCRIPTION
**Pull request by taylonr**
I added a new **Getting Started** section to the `readme.md` file. I grabbed the data from the wiki combined with what I did this morning to include it into a project. I think having a spot on the readme makes it easier to find, and has become somewhat expected with other projects showing the base use case of getting up and running.

image

### CODE REVIEW

**Comment by taylonr** [comment]
@ThomasBurleson yeah, nothing I wrote was new. It was more just consolidated. I took the npm commands from the wiki, the code example from one of the demo apps.

I was more trying to see if there was a way that was more....obvious.

For some perspective, I googled "angular flexbox" and this was one of the top repos. When the readme loaded I saw `Quick Links` and thought "Well, I don't want chat or forums, I'm not yet where I need the wiki."

So I kept scrolling. Then I assumed the `Developers` section was for folks who want to develop more features for the repo, not developers using the library.

I checked out the `Demos` and that showed me some how to use it. So then I finally went through the wiki and found the npm section.

I glanced at the `Browser Support` but for the most part, _I_ didn't care about it, because I knew the browser support for Flexbox, I'd used it with other frameworks.

So I added a section to the readme for the `Getting Started` as a "If you know you want this and want to start playing with it" guide.

There might be other options, like renaming the `Wiki Documentation` to `Getting Started.` Although, personally, I think even the landing page there could use some help to guide people in how to get started quickly.

File: README.md [hunk 1]
```diff
@@ -1,21 +1,52 @@
...
+complexities and workarounds encountered with the traditional, manual, CSS-only application of box CSS.
 
-The **real** power of Flex Layout, however, is its **responsive** engine. The 
-[Responsive API](https://github.com/angular/flex-layout/wiki/Responsive-API) enables developers to easily specify 
+The **real** power of Flex Layout, however, is its **responsive** engine. The
+[Responsive API](https://github.com/angular/flex-layout/wiki/Responsive-API) enables developers to easily specify
 different layouts, sizing, visibilities for different viewport sizes and display devices.
 
+---
+### Getting started
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
### Getting Started
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,52 @@
...
+
+Start by installing the Angular Flex Layout library from `npm`
+
+`npm i -s @angular/flex-layout@latest @angular/cdk@latest`
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
`npm i @angular/flex-layout @angular/cdk`
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,52 @@
...
+
+Next, you'll need to import the flex-layout module in your app's module.
+
+**app.module.ts**
+
+```
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
```ts
```

File: README.md [hunk 4]
```diff
@@ -1,21 +1,52 @@
...
+
+@NgModule({
+    ...
+    imports: [ FlexLayoutModule ],
+    ...
+});
+```
+
+After that is configured, you can use the attributes on your HTML tags for flex layout:
+```
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
```html
```

File: README.md [hunk 5]
```diff
@@ -1,21 +1,52 @@
...
+...
+
+@NgModule({
+    ...
+    imports: [ FlexLayoutModule ],
+    ...
+});
+```
+
+After that is configured, you can use the attributes on your HTML tags for flex layout:
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
After that is configured, you can use the Angular Layout attributes in your HTML:
```

File: README.md [hunk 6]
```diff
@@ -1,21 +1,52 @@
...
+```
+<div fxLayout="row" fxLayoutAlign="space-between">
+</div>
+```
+
+Check out [all the available options](https://github.com/angular/flex-layout/wiki/Declarative-API-Overview) for using layout in your application.
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
Check out [all of the available options](https://github.com/angular/flex-layout/wiki/Declarative-API-Overview) for using Angular Layout in your application.
```

File: README.md [hunk 7]
```diff
@@ -1,21 +1,52 @@
...
 different layouts, sizing, visibilities for different viewport sizes and display devices.
 
+---
+### Getting started
+
+Start by installing the Angular Flex Layout library from `npm`
+
+`npm i -s @angular/flex-layout@latest @angular/cdk@latest`
+
+Next, you'll need to import the flex-layout module in your app's module.
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
Next, you'll need to import the Layout module in your app's root module.
```

File: README.md [hunk 8]
```diff
@@ -1,21 +1,52 @@
...
-The **real** power of Flex Layout, however, is its **responsive** engine. The 
-[Responsive API](https://github.com/angular/flex-layout/wiki/Responsive-API) enables developers to easily specify 
+The **real** power of Flex Layout, however, is its **responsive** engine. The
+[Responsive API](https://github.com/angular/flex-layout/wiki/Responsive-API) enables developers to easily specify
 different layouts, sizing, visibilities for different viewport sizes and display devices.
 
+---
+### Getting started
+
+Start by installing the Angular Flex Layout library from `npm`
```
Comment (CaerusKaru) [inline README.md]:
```suggestion
Start by installing the Angular Layout library from `npm`
```

**Comment by taylonr** [comment]
@CaerusKaru implemented your suggestions.

**Comment by CaerusKaru** [comment]
@taylonr Thanks for your work on this!

**Pull request merged**